.github/workflows/ci_tests_dev.yaml

```diff
@@ -133,14 +132,6 @@ jobs:
 
       # Pull baseline image data from dvc remote (DAGsHub)
       - name: Pull baseline image data from dvc remote
```

Comment:
Lines 134-139 and lines 141-146 are duplicated.

Reply:
Yeah, looks like we forgot to remove these lines in #2045.